### ISSUE TITLE
Emissions Accounting of Synthetic Fuels / (short-term) CCU

### ISSUE THREAD

**Issue by fschreyer** [IAMconsortium/common-definitions#49]
Dear all,

there is a general confusion in IAM projects about where to account for carbon contained in synthetic fuels produced via CCU. The emissions variable definitions do not specify where to account for these emissions and there are some intricacies with the different options. I will try to describe the three options that exist and which one I prefer and why. 

### Options

Let's take the example that we capture some CO2 from an industry plant and use it to produce a synthetic liquid fuel e.g. via Fischer-Tropsch using electrolytic hydrogen and combust this synthetic fuel in the transport sector. There are three levels at which the positive/negative emissions could be accounted:

1.) **CO2 provider:** This would mean that the CO2 is accounted as emissions for industry, while to the synfuel producer (energy supply) sector and the synfuel user (transport sector) no emissions are accounted. Note that this would apply to both fossil and non-fossil CO2. If it was fossil CO2 that was captured, the industry plant would still have the full positive emissions, while if it was, for example, bioenergy the industry plant would have zero emissions instead of negative emissions (which it would get if it stored the carbon underground). 

2.) **Supply-side:** This would account the carbon as emissions of the synfuel producer. It would mean that the CO2 is accounted as emissions for energy supply, while it would not account emissions (or account negative emissions) for industry and would neither account emissions for transport. 

3.) **Demand-side:** This would account the carbon as emissions of the synfuel user. It would mean that the CO2 is accounted as emissions for transport, while there would be no emissions for the synfuel-producer and no emissions for industry (if CO2 is fossil) or negative emissions (if CO2 is non-fossil). 

**I would definitely want to include a short statement in the definition of emissions variables on where to account this CO2 because it is not that obvious from the IPCC emissions guidelines. I am generally in favor of option 1, see my arguments below.** 

### Arguments

a) What would be desirable, of course, is if we had the same definition as the [IPCC emissions accounting guidelines](https://www.ipcc-nggip.iges.or.jp/public/2006gl/). However, I am not sure the 2006 guidelines had this fully thought through and also the [2019 update](https://www.ipcc-nggip.iges.or.jp/public/2019rf/index.html) did not seem to have any major update on this, if I see it right. The guidelines say

> "Should CO2 capture technology be installed and used at a plant, it is good practice to deduct the CO2 captured in
a higher tier emissions calculation. Quantities of CO2 for later use and short-term storage should not be deducted
from CO2 emissions except when the CO2 emissions are accounted for elsewhere in the inventory[4]. The default assumption is that there is no carbon dioxide capture and storage (CCS) taking place.5" (IPCC Guideline 2006, Vol. 3, Chapter 1, p. 17-18)." 

So, this sounds like option 1.) However, the footnote with the exceptions where CO2 emissions are accounted elsewhere points e.g. to methanol production. It says

> "Petrochemical processes may utilise CO2 captured elsewhere as a feedstock, and CO2 may also be captured from
petrochemical processes. This may create potential double counting issues. For example, some methanol plants may utilise by-product CO2 captured from other industrial processes as a feedstock for methanol production. To avoid double counting the CO2 captured should not be reported as CO2 emissions from the process from which
the CO2 is captured." (IPCC Guideline 2006, Vol. 3, Chapter 3, p. 357)

To me this sounds like an exception for methanol to account under approach 2), which would not make sense if it was only about this specific product. But maybe it only applies to by-product CO2 from petrochemical processes which CCU + green H2 would not be but in reality this will be hard to distinguish. To be fair, the 2006 guidelines date quite some time back now where the synfuel story was not yet as relevant. 

In summary, I find the IPCC guidelines not completely clear on this, but they tend to support the logic of approach 1.). 

b) The EU ETS is going in the direction of approach 1.). Capturing CO2 only frees you from submitting allowances if you show that they are stored. Moreover, a recent ETS directive specifically pointed out again that synthetic fuels should be regarded as emissions-free to aircraft operators in the aviation sector, that is, exclude approach 3.). 

c) Thinking about the communication of IAM results to the broader public, I would also prefer approach 1.). This is because of this frequent criticism that IAMs tend to overemphasize CDR.  We would stress this at first glance even more in sectoral emissions plots with options 2.) and 3.) as there will be more "negative emissions" which are actually not CDR but just carbon used in other sectors.

d) Finally, a counterargument or at least a disclaimer to option 1.). It leads to some definition issues for gross emissions and CDR variables in my view.  Typically, we would want to define gross emissions to fulfill gross emissions + CDR = net emissions and gross emissions should always be greater or equal 0. However, so far the gross emissions only exclude CO2 captured from bioenergy not from synthetic fuels (which under option 1. would also be emissions-free on the demand-side). Including non-fossil CO2 in gross definition can still lead to the situation, though, that e.g. industry gross emissions can become negative, if somebody captured fossil CO2, produced synthetic fuel with it and industry would capture and store it. @strefler and I already had some discussion about that. I would want to create a separate issue on that. There are potentially more of these intricacies I am not aware of; maybe also with other options. 

Let me know what you think. 

Best,
Felix

@IAMconsortium/common-definitions-emissions

**Comment by jayfuhrman** [comment]
This is an excellent point and definitely worth clarifying in the top level of the Emissions|CO2 variable category.  To me I think it comes down to a choice between being more consistent with some policy frameworks (e.g., in the EU) for biofuels carbon accounting (option 1), or with the physical reality of where emissions or removals are actually occurring. 

I'm slightly in favor of option 3 but would defer to others as well

**Comment by fschreyer** [comment]
I agree that the physical accounting logic would, generally, be the preferred one but I think it would get quite confusing when we relate to the current policy discourse.

We talked about this in some internal discussions again and would make the following suggestion:

1) We account synfuel emissions on the side of the CO2 Capture (option 1). Biofuels and synfuels are closely related (in fact, you can argue that 2nd gen. biofuels are a form of synfuels) such that it makes sense to treat them equally. Moreover, the policy consistency is important. One additional example is that considering future trade of synfuels, accounting for physical carbon flows would make regional net-zero goals (e.g. in the EU) much harder to reach as imported synfuels will not be accounted carbon-neutral. For analysis of regional emissions goals, we would not like to switch to a physical accounting which does not correspond to the accounting typically used in policy analysis. 

2) Given 1), there is a specific issue with the treatment of fossil-based synfuels and the definition of gross emissions and carbon removals. If the fossil carbon is obtained via CO2 capture, it has already been accounted as emissions in the CO2 capture sector, such that it should not be accounted another time when it is released to the atmosphere. That means it needs to be accounted as negative emissions for some sector once if the CO2 is again captured and permanently stored (see illustration below). However, these negative emissions are not carbon removal as it is fossil CO2. I would therefore propose to introduce a separate category like ``Emissions|CO2|Sector C|Accounted in Other Sectors`` that we also deduce from net emissions when we calculate gross emissions of a sector. That is,

``Gross Emissions`` = ``Net Emissions`` - ``Carbon Removal`` - ``Emissions Accounted in Other Sectors``.

Another option would be to argue that these emissions should not be substracted from the net emissions of sector C but of the original sector A. However, that would require to track the origin of all carbon flows through the system which seems hardly feasible once there are many closed loops / consecutive usages of carbon. 

![image](https://github.com/IAMconsortium/common-definitions/assets/42831977/3db7a6c8-9ec1-4736-b40e-3d22a5634b73)

3.) The concept needs to be consistent with accounting of carbon in materials, more generally. That is, the question where emissions of chemical feedstocks / products / waste are accounted. This accounting deserves opening a separate issue but as soon as carbon accounting methods are chosen which do not align with physical carbon flows, the same problems arise, which should be solved in the same way. 

This solution in 2.) is a little tedious, I admit, but it is an edge case, which allows us to stay with a more policy-relevant accounting. 

Do you have concerns about this?
